### PR TITLE
update!: Remove PrivacyBlur

### PR DESCRIPTION
**Pull request by redoomed1**
List of changes proposed in this PR:

- Remove PrivacyBlur from the Data Redaction tool recommendations, as the GitHub repository for the app appears to be deleted, which means that the source code is inaccessible
  - Relevant discussion: https://discuss.privacyguides.net/t/remove-privacyblur/11560/22
  - PrivacyBlur's website, which appears to be hosted on GitHub Pages, is also inaccessible.
  - I'm not sure where to put the admonition that warns about blurring text in images since its fairly out of place on a page where the rest of the tools are metadata scrubbers. There's already guidance on the site about how to [black out information on iOS](https://www.privacyguides.org/en/os/ios-overview/#blacking-out-facesinformation), so maybe the warning can find a place in the Android Overview or General Apps (for Android) page[^1]?

[^1]: Given that PrivacyBlur is only available for Android and iOS

### CODE REVIEW

File: docs/data-redaction.md [hunk 1]
```diff
@@ -11,6 +11,15 @@ cover: data-redaction.webp
 
 When sharing files, be sure to remove associated metadata. Image files commonly include [Exif](https://en.wikipedia.org/wiki/Exif) data. Photos sometimes even include GPS coordinates in the file metadata.
 
+</div>
+
```
Comment (friadev) [inline docs/data-redaction.md]:
```suggestion
```

Comment (friadev) [reply docs/data-redaction.md]: Accidentally added a closing </div>

**Review by friadev — CHANGES_REQUESTED** [review]

**Review by friadev — APPROVED** [review]

**Review by friadev — APPROVED** [review]

File: docs/data-redaction.md [hunk 2]
```diff
@@ -11,6 +11,13 @@ cover: data-redaction.webp
 
 When sharing files, be sure to remove associated metadata. Image files commonly include [Exif](https://en.wikipedia.org/wiki/Exif) data. Photos sometimes even include GPS coordinates in the file metadata.
 
+<div class="admonition warning" markdown>
+<p class="admonition-title">Warning</p>
+
+You should **never** use blur to redact [text in images](https://bishopfox.com/blog/unredacter-tool-never-pixelation). If you want to redact text in an image, draw a box over the text. For this, we suggest apps like [Pocket Paint](https://github.com/Catrobat/Paintroid).
```
Comment (jonaharagon) [inline docs/data-redaction.md:17]: I feel like I brought this up on the forum at one point, but Pocket Paint should just be an actual recommendation if we are going to recommend it like this.

**Review by jonaharagon — APPROVED** [review]

File: docs/data-redaction.md (hunk 2 above)
Comment (friadev) [reply docs/data-redaction.md:17]: Good point

Comment (redoomed1) [reply docs/data-redaction.md:17]: I can remove the last sentence recommending it, and we can evaluate Pocket Paint separately in the [existing thread](https://discuss.privacyguides.net/t/paintroid-pocket-paint/16804).

Comment (friadev) [reply docs/data-redaction.md:17]: Good idea I think

**Pull request merged**